_includes/manuals/2.3/1.2_release_notes.md

```diff
@@ -0,0 +1,23 @@
+## Release notes for {{page.version}}
+
+This section will be updated prior to the next release.
+
+### Headline features
+
+### Release Notes
+
+### Deprecations
+
+### Upgrade warnings
+
+* Index and column named `digest` on SQL tables `messages` and `sources` will be dropped during `db:migrate` step and replaced by more effective hash index on column named `value`. While we are working on redesigning how reports are stored in the SQL database, hash indices are faster and use significantly less memory storage than btree in this case. Our tests indicate that on a standard VM, index creation rate is at 18 million records per minute, this step should be from seconds to several minutes depending on size of the tables.
+
+* Report creation via `/api/v2/config_reports/` REST POST no longer returns the full report for performance reasons. Only basic metadata like database ID is returned, when full report is needed query the same REST resource via HTTP GET method to fetch it. More info at https://github.com/theforeman/foreman/pull/7987
```

Comment:
This part should be cleaned from ` _includes/manuals/nightly/1.2_release_notes.md` as well

Reply:
Could you also add new note for: https://github.com/theforeman/foreman/pull/8083/files

The following fact names are ignored by default due to high number of changes: load_averages, memory::system and memory::swap (capacity, used, available names). Ignored facts can be changes in Administer - Settings - Provisioning.

Reply:
Added this in upgrade_warnings, when we will add release notes for 2.3 it will be added in Facts section, so I think we can remove it later or you want it to keep it later also ?